package.json

```diff
@@ -5,15 +5,15 @@
   "main": "src/index.js",
   "repository": "https://github.com/Avaq/node-server-skeleton.git",
   "scripts": {
-    "start": "node_modules/.bin/babel-node ./",
-    "dev": "DEBUG=* node_modules/.bin/babel-node node_modules/.bin/node-dev ./",
-    "lint": "node_modules/.bin/eslint ./src ./test",
-    "pretest": "npm run lint",
-    "test": "npm run version && npm run clean && npm run test:unit && npm run test:integration",
-    "test:unit": "NODE_ENV=test node_modules/.bin/babel-node node_modules/.bin/isparta cover --root ./src --report text --report html test/index.js -- test/unit",
-    "test:integration": "NODE_ENV=test node_modules/.bin/babel-node test/index.js test/integration",
-    "setup": "npm run version && npm install && npm dedupe && npm run test:unit && npm outdated",
-    "clean": "node_modules/.bin/rimraf npm-debug.log coverage",
+    "clean": "rimraf npm-debug.log coverage",
+    "dev": "DEBUG=* babel-node node_modules/.bin/node-dev .",
+    "lint": "eslint src test",
+    "pull": "git pull && npm run version && npm install && npm dedupe && npm outdated",
+    "push": "npm test && git push && git push --tags",
```

Comment:
Out of curiosity. Can you explain the idea behind using npm for git commands and how you would integrate this in your workflow?